embulk-core/src/main/java/org/embulk/jruby/LazyScriptingContainerDelegate.java

```diff
@@ -315,6 +317,7 @@ synchronized ScriptingContainerDelegateImpl getInitialized() {
                     this.classLoader, this.delegateLocalContextScope, this.delegateLocalVariableBehavior);
             if (this.initializer != null) {
                 this.initializer.initialize(this.impl);
+                this.logger.info(String.format("Loaded JRuby %s Compatible MRI: %s",this.impl.getJRubyVersion(),this.impl.getRubyVersion()));
```

Comment:
* You can use `"{}"` for string replacement in SLF4J loggers.
* The "Ruby version" from JRuby is often confusing. Let's just have the JRuby version.

```suggestion
                this.logger.info("Loaded JRuby runtime {}", this.impl.getJRubyVersion()));
```

Reply:
Thanks! fixed.

The output MRI version helps check the RubyGems site.
I don't know any website or document mapping JRuby and MRI versions.

<img width="1047" alt="スクリーンショット 2022-12-09 13 25 51" src="https://user-images.githubusercontent.com/767650/206624746-eecb8d92-46a4-4b63-b2d9-fa7017cda0c7.png">

Reply:
It's usually mentioned in their release notes : 
* "Compatibility is updated to Ruby 3.1 level." - https://www.jruby.org/2022/11/23/jruby-9-4-0-0.html
* "JRuby 9.3.x is compatible with Ruby 2.6.x and stays in sync with C Ruby." - https://www.jruby.org/2022/10/24/jruby-9-3-9-0.html

But, the information is sometimes not perfectly accurate. Some recent Ruby features might be implemented earlier, and some might delay. Even if we show "Compatible Ruby version", that can be wrong. Users would ask us "It should it's 3.X.X-compatible, but does not work!", and we have to investigate by ourselves... I'd like to avoid that.

As the Embulk project, I don't have intention (and resources) to take care of JRuby versions. Choosing an appropriate JRuby version would be user's responsibility. Users have responsibility to look into https://www.jruby.org/ by themselves, and to think which version they should use.

Reply:
Thanks! I'll create a compatible table by myself. (For personal use)